src/Modules/core.ts

```diff
@@ -185,14 +245,17 @@ export class CoreModule extends BaseModule {
                 Player.LSCG = <SettingsModel>{};
                 this.registerDefaultSettings();
             }
-            previousVersion = Player.LSCG.Version = LSCG_VERSION;
+            Player.LSCG.Version = LSCG_VERSION;
             saveRequired = true;
         }
-        saveRequired = saveRequired || this.CheckForMigrations(previousVersion);
+        saveRequired = this.CheckForMigrations(previousVersion) || saveRequired;
```

Comment:
Might want to reconsider this change.

 `previousVersion` can be null/undefined at this point for new accounts if (which sets `saveRequired == true`), triggering a type error down below in `CheckForMigrations()`:
```
Uncaught TypeError: fromVersion is undefined
    CheckForMigrations core.ts:261
    CheckVersionUpdate core.ts:251
    init main.ts:122
    LSCG main.ts:59
    // etc.
```

Reply:
Issue was if saveRequired is true from the version update, it would short-circuit and never run migrations. Will add a skip if fromVersion is missing in CheckForMigrations.